### ISSUE TITLE
vertical scroll bar missing for font and open file dialog

### ISSUE THREAD

**Issue by rohrlich** [goki/gi#221]
For some dialogs the vertical scrollbar is missing, i.e. not visible, but you can get it with window resizing. However, and this is important, as evidenced by the ScrollDelta func the scrollbar is still not really there because HasScroll[Y] is false. HasScroll[Y] is only true when the window is made big enough to not require a scrollbar.  John will follow up unless someone else just happens to know what is wrong.

**Comment by rohrlich** [comment]
Clarification - HasScroll[Y] is true if the window is wide enough to not require the horizontal scrollbar but when both are visible HasScroll[Y] is false. This is not true for all windows, some register both HasScroll[Y] and HasScroll[X] as true when both are visible.

**Comment by rohrlich** [comment]
This seems to be a drawing problem rather than a logic problem. Best for Randy to take a look.
<img width="175" alt="screen shot 2018-11-25 at 5 47 07 pm" src="https://user-images.githubusercontent.com/22350787/48988422-4a024680-f0da-11e8-9e25-1ac9b4c17447.png">

**Comment by rohrlich** [comment]
I changed the title as I think that fixing the more general case might resolve the specific issue first mentioned.

**Comment by rohrlich** [comment]
Yes, I think you are correct for the case you mention. The inner element has its own scroll bar which may scroll out of view while the outer frame scroll bars should always be visible if needed. When I first filed the ticket I was looking at the font selection dialog which because it is the only element seems like it is the outermost frame and should always have scrollbars. Users will probably not think, oh this is a tableview...  I certainly don't want to break stuff that is working so I guess we should leave well enough alone for now  but there are other cases, consider the gide prefs window itself, which has the same problem. Is it a tableview? And that window also has the problem of not scrolling vertically with trackpad but does scroll vertically using the scroll bar. I do realize that I made the window needlessly small to test this and with the small number of prefs it wouldn't be an issue for this particular window but it could be for ones with more elements.
